package.json

```diff
@@ -10,19 +10,18 @@
   "type": "module",
   "scripts": {
     "build": "tsc -b && vite build",
+    "build-storybook": "storybook build",
+    "check": "tsc -b",
     "copy:arcgis": "cpy \"./node_modules/@arcgis/core/assets/**\" ./public/assets",
-    "dev": "npm start",
-    "dev:vite": "wait-on http://127.0.0.1:4000 && vite",
     "dev:firebase-emulators": "firebase emulators:start --import emulator_data --export-on-exit --only functions,auth,ui,firestore",
+    "dev:vite": "wait-on http://127.0.0.1:4000 && vite",
     "format": "prettier . --write",
     "lint": "eslint . --report-unused-disable-directives --max-warnings 0",
-    "check": "tsc -b",
     "preview": "vite preview",
-    "start": "run-p dev:*",
+    "start": "concurrently 'npm run dev:vite' 'npm run dev:firebase-emulators'",
```

Comment:
**question**: oh fun, what made you switch?

Reply:
run-p seemed like it had issues shutting down the firebase emulators. I always had to press control-c twice. Then I looked it up on npm and it hasn't had a new release in something like six year so I tried something more current (pun intended).